applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -584,7 +602,8 @@ fetch_local_resources(AccountId) ->
         {'ok', JObjs} ->
             CacheProps = [{'origin', fetch_local_cache_origin(JObjs, AccountDb, [])}],
             Resources = resources_from_jobjs([wh_json:get_value(<<"doc">>, JObj) || JObj <- JObjs]),
-            LocalResources = [Resource#resrc{global='false'} || Resource <- Resources],
+            Proxies = fetch_account_dedicated_proxies(AccountId),
```

Comment:
Could we pass Proxies into `resources_from_jobjs/2` and set them there, rather than building the list of resources, followed by iterating over the list again to set the proxies property?